supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -66,6 +66,7 @@ object PipelinesApiAsyncBackendJobExecutionActor {
   val JesPreemption = 14
 
   val PapiFailedPreConditionErrorCode = 9
+  val PapiErrorCode10 = 10
```

Comment:
Can we name this something more descriptive, `PapiPreemptionErrorCode` perhaps?

Reply:
I think the whole problem is that 10 is supposed to mean "crashed" but is being used to mean "pre-empted"?

Reply:
Ok, `PapiCrashedErrorCode` perhaps?

Reply:
Sure! I can rename it as suggested